main.tf

```diff
@@ -67,15 +25,9 @@ module "container_definition" {
 
   environment = merge({
     # General Settings
-    EXTERNAL_PORT_HTTPS = var.external_port
-    INTERNAL_PORT_HTTPS = var.internal_port
-    EXTERNAL_PORT_HTTP  = var.external_port
-    INTERNAL_PORT_HTTP  = var.internal_port
-    USE_TLS_AUTH        = false
-
     LLM_CLASSIFIER_URL = var.deploy_llm ? "http://llm-classifier:${var.llm_classifier_port}" : null
 
-    # JWT
+    # JWT Auth Settings
```

Comment:
Would make sense IMO to also include the variables `SOMBRA_REVERSE_TUNNEL_USE_HTTPS`, and `EXTERNAL_PORT_HTTP` to allow the [script](https://github.com/transcend-io/main/blob/01cd1d34da4ca902aa5d107020556c1ac4834ac9/backend-services/sombra/config/piko/lib.js#L70) to figure out what settings it needs to use. Those options still need to be documented in the docs though